src/shared/components/DataQueryComponent/DataQueryComponent.tsx

```diff
@@ -81,6 +81,7 @@ const DataQueryComponent: FunctionComponent<DataQueryComponentProps> = ({
 										'shared/components/data-query-component/data-query-component___er-ging-iets-mis-tijdens-het-ophalen'
 									)}
 									icon="alert-triangle"
+									actionButtons={['home']}
```

Comment:
Ik zou deze hier niet standaard zetten aangezien deze component ook in het beheer gebruikt wordt